packages/csi-wekafs/csi-wekafs.patch

```diff
@@ -0,0 +1,60 @@
+diff -x '*.tgz' -x '*.lock' -uNr packages/csi-wekafs/charts-original/app-readme.md packages/csi-wekafs/charts/app-readme.md
+--- packages/csi-wekafs/charts-original/app-readme.md
++++ packages/csi-wekafs/charts/app-readme.md
```

Comment:
The contents of `app-readme.md` and `questions.yaml` shouldn't be in the patch. They are and should be in the overlay directory which is basically where you keep files that you want added in the resulting tgz in assets/csi-wekafs once it gets generated. Please delete them from `packages/csi-wekafs/charts` after the make prepare step. They will then get automatically removed from the patch once you do the make patch step after.